load.go

```diff
@@ -177,3 +181,14 @@ func readFile(templateKey string, baseDir string, filePath string) string {
 	return string(dat)
 
 }
+
+func (m Mock) patchedWith(patch Mock) *Mock {
+	baseStruct := structs.New(&m)
+	patchStruct := structs.New(patch)
+	for _, field := range patchStruct.Fields() {
+		if !field.IsZero() {
+			baseStruct.Field(field.Name()).Set(field.Value())
```

Comment:
This also changes the Key, right? We should keep the baseStruct's key.

Reply:
I wonder if explicitly setting each field is better.

Reply:
it takes the Key from the "patch" behavior, the one with the `include` property, which is what we want

Reply:
nvm, you're right. patchStruct includes baseStruct.